examples/flutter_gallery/lib/demo/pesto_demo.dart

```diff
@@ -569,266 +572,146 @@ class RecipeStep {
 
 const List<Recipe> kPestoRecipes = const <Recipe>[
   const Recipe(
-    name: 'Pesto Bruschetta',
+    name: 'Roasted Chicken',
     author: 'Peter Carlsson',
-    ingredientsImagePath: 'food/icons/quick.png',
+    ingredientsImagePath: 'pesto/main.png',
     ingredientsImagePackage: _kGalleryAssetsPackage,
-    description: 'Bask in greens this season by trying this delightful take on traditional bruschetta. Top with a dollop of homemade pesto, and season with freshly ground sea salt and pepper.',
-    imagePath: 'food/image1.jpg',
-    imagePackage: _kGalleryAssetsPackage,
+    description: 'Establish your superiority to your friends and family with the juiciest roasted chicken they\'ll ever savor. Pair with roasted veggies to declare yourself the Roast Champion.',
+    imagePath: 'assets/image1.png',
+//    imagePackage: _kGalleryAssetsPackage,
     ingredients: const<RecipeIngredient>[
-      const RecipeIngredient(amount: '6 pieces', description: 'Mozzarella cheese'),
-      const RecipeIngredient(amount: '6 pieces', description: 'Toasts'),
-      const RecipeIngredient(amount: '⅔ cup', description: 'Homemade pesto'),
+      const RecipeIngredient(amount: '1 whole', description: 'Chicken'),
+      const RecipeIngredient(amount: '1/2 cup', description: 'Butter'),
+      const RecipeIngredient(amount: '1tbsp', description: 'Onion powder'),
```

Comment:
I know the existing references to tablespoons don't have a space between the quantity and the volume but I think we should add in a space.